resolwe/flow/models/utils.py

```diff
@@ -112,8 +113,10 @@ def validate_file(field, regex):
 
         if path_prefix:
             path = os.path.join(path_prefix, filename)
-            if not os.path.isfile(path):
+            if not os.path.exists(path):
                 raise ValidationError("Referenced file ({}) does not exist".format(path))
+            if not os.path.isfile(path):
+                raise ValidationError("Referenced file ({}) is not a normal file".format(path))
```

Comment:
Do we want to enforce that? I think it should be ok to reference a link, even though i don't see a use-case for this.